hnn_core/extracellular.py

```diff
@@ -28,6 +28,30 @@
 from .externals.mne import _validate_type, _check_option
 
 
+def calculate_csd2d(data, delta=1):
```

Comment:
make this a private method for now. You can always make it public if we want to expose it to users.